portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,9 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomQuote() {
+    // TODO store all possible quotes from excel file
```

Comment:
You can probably just store this in an array in its own javascript file. No need to overcomplicate things.